### ISSUE TITLE
Reload config while running

### ISSUE THREAD

**Issue by Vosjedev** [directvt/vtm#534]
I find it a bit annoying to have to restart vtm when I modified the config. Maybe it is possible to reload the config while running?\

(This can be at the bottom of the checklist, please finish what you want to finish earlier first.)

**Comment by o-sdn-o** [comment]
Yes, this will definitely be done. This will be done together with Settings UI - all options from the settings can be configured through the UI. The only difficulty there is that the configuration is loaded from several files and plus a piece arrives via DirectVT, and we need to figure out how to react to updating configuration sources.
